tiatoolbox/dataloader/save_tiles.py

```diff
@@ -0,0 +1,59 @@
+"""Save image tiles from the whole slide image"""
+from tiatoolbox.dataloader import wsireader
+from tiatoolbox.decorators.multiproc import TIAMultiProcess
+from tiatoolbox.utils import misc
+
+
+@TIAMultiProcess(iter_on="input_path")
+def save_tiles(
+    input_path,
+    output_dir="tiles",
+    tile_objective_value=20,
+    tile_read_size_w=5000,
+    tile_read_size_h=5000,
+):
+    """Save JPEG image tiles for whole slide image. Multiprocessing decorator runs this
```

Comment:
Not only saving JPEG tiles - JPEG is default, but user can specify image file format.

Reply:
Addressed in this commit https://github.com/TIA-Lab/tiatoolbox/commit/1b0770399a4fdc7eb88ddf1cfb4aa320ee7e2dfc